addons/website/static/src/components/switch/switch.js

```diff
@@ -15,7 +15,7 @@ export class Switch extends Component {
     };
     static template = xml`
     <label t-att-class="'o_switch' + extraClasses">
-        <input type="checkbox" t-att-checked="props.value" t-att-disabled="props.disabled" t-on-change="(ev) => props.onChange(ev.target.checked)"/>
+        <input type="checkbox" class="visually-hidden" t-att-checked="props.value" t-att-disabled="props.disabled" t-on-change="(ev) => props.onChange(ev.target.checked)"/>
```

Comment:
There are problems with this:
- First, the previous commit about "enter on single input" was already dubious: you enable it for the edit menu dialog and for the add page dialog... both not having a single input. I am not sure this is good.
- Combined with this commit , this makes the add page dialog switch weird: you can select it with the keyboard but not uncheck it with your enter key as it validates the dialog.
- Without the previous commit, I can't seem to check/uncheck that switch with my keyboard at all. So making it focusable only make seems worse in the current state :grimacing: